cli/sdk/src/sendPanic.ts

```diff
@@ -64,7 +64,7 @@ export async function sendPanic(
       platform: await getPlatform(),
       liftRequest: JSON.stringify(error.request),
       schemaFile: maskedSchema,
-      fingerprint: getFid() || undefined,
+      fingerprint: getFid(),
```

Comment:
We can directly call `checkpoint.signature.sync()` here

Reply:
I was thinking about that too :)